app/assets/stylesheets/mixins/vars.sass

```diff
@@ -10,15 +10,6 @@ $tablet-width: 740px
 $laptop-width: 960px
 $desktop-width: 1300px
 
-$dark-gray: #7b7b7b
-$dark-gray-d: #848585
-$gray: #d7d7d6
-$light-gray: #9d9d9c
-$blue: #c5e0dc
-$light-blue: #ebf4f3
-$orange: #e08e79
-$gray-d: #dcdcdc

```

Comment:
These variables are a carry over from the old project, and were only in here as a deprecation.

They should be replaced by the variables below, whichever is specific for the situation. I've removed and replaced all these variables.